doc/source/configure.rst

```diff
@@ -135,9 +135,31 @@ Head Node
 In addition to ports specified above, the head node needs to open several more ports.
 
 - ``--port``: Port of GCS. Default: 6379.
-- ``--dashboard-port``: Port for accessing the dashboard. Default: 8265
+- ``--redis-shard-ports``: Comma-separated list of ports for non-primary Redis shards. Default: Random values.
 - ``--gcs-server-port``: GCS Server port. GCS server is a stateless service that is in charge of communicating with the GCS. Default: Random value.
 
+- If ``--include-dashboard`` is true (the default), then the head node must open ``--dashboard-port``. Default: 8265.
+
+If ``--include-dashboard`` is true but the ``--dashboard-port`` is not open on
```

Comment:
Thanks for the PR!! It is really helpful :).

Btw, this is probably something we should resolve (although it makes sense to document now as it is not working now). @mfitton Do you have any idea why this happens?

Reply:
I don't know why it happens. I'll create an item to look into it next sprint.